.circleci/config.yml

```diff
@@ -343,47 +343,58 @@ workflows:
     jobs:
       - checkout:
           <<: *default-context
+          name: 'Checkout'
       - test_unit:
           <<: *default-context
+          name: 'JSDOM tests'
```

Comment:
This was nice before to train us to know that `pnpm test:unit` runs this one.

Reply:
We would usually check the circleci log to see why it failed, and the step will tell you the exact command to run. Both our browser and jsdom tests are unit tests, so it is a bit misleading.

For context, this is based on the base-ui changes, which we agreed on code-infra to bring to other repos: https://github.com/mui/base-ui/pull/1321

Reply:
Personally prefer the existing naming, but I value consistency more 🙂

Reply:
> This was nice before to train us to know that pnpm test:unit runs this one.

In Base UI this script was also renamed to `test:jsdom`, so it's still consistent.

Reply:
I think we should rename those to `test:unit:jsdom` and `test:unit:browser`, to show that it's running the same thing in different environments, and change the titles accordingly.

Reply:
And I also feel that having the job names map to npm scripts like we currently do is more convenient.